nautobot/ipam/utils/migrations.py

```diff
@@ -1,46 +1,84 @@
 import collections
 import sys
+from time import monotonic
 
 from django.core.exceptions import ValidationError
 from django.db import models
 import netaddr
 
+from nautobot.ipam.constants import IPV4_BYTE_LENGTH, IPV6_BYTE_LENGTH
+
 BASE_NAME = "Cleanup Namespace"
 DESCRIPTION = "Created by Nautobot 2.0 IPAM data migrations."
 GLOBAL_NS = "Global"
 
 
+class TimerContextManager:
+    def __init__(self, message, indent=""):
```

Comment:
Just a suggestion and definitely not blocking. It would be nice if this could also be used as a decorator and we could decorate the `process_namespaces` function to get a timer for the full process.